src/components/Form/UserResourceForm.vue

```diff
@@ -36,23 +38,24 @@ const numberFieldRules = useNumberFieldRules();
 const selectedUser = ref("");
 const quantity = ref("");
 
-const userOptions = computed(() =>
-  store.getters["users/allUsers"].map((user) => user.name)
-);
+const allUsers = computed(() => store.getters["users/allUsers"]);
 
 if (route.path.includes("/remove")) {
   const resourceId = route.params.resourceId;
   const affectedUserId = route.params.userId;
-  
+
   const affectedUser = computed(() =>
     store.getters["users/getUserById"](affectedUserId)
   ).value;
 
-  selectedUser.value = affectedUser.name;
+  selectedUser.value = affectedUser.firstName;
```

Comment:
Is selectedUser supposed to be just the `firstName` here? I see in the `handleSubmit` method selectedUser is considered an object because we are doing `selectedUser.value.id`.

Reply:
fixed